studioclass-backend/src/graphql/mutations/posts.js

```diff
@@ -70,7 +69,13 @@ export const resolvers = {
       await newPost
         .populate([{ path: 'user', select: 'id username'}, { path: 'tags' }]);
 
-      pubsub.publish('POST_ADDED', { postAdded: newPost });
+      console.log(newPost);
```

Comment:
On the backend, use a logging lib like winston on pinon to capture stuff like this.

Also, maybe you don't need to log here. That can get expensive.